ansible/group_vars/all

```diff
@@ -85,6 +85,8 @@ db:
   authkeys:
   - guest
   - whisk.system
+  whisk:
+    actions: "{{ db_prefix }}whisks"
```

Comment:
If this does in fact work (I expect it not too): Can we move all database name definitions here?